lang/cs/Org.Apache.REEF.Common/Files/ResourceHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.Reflection;
+using System.Resources;
+
+namespace Org.Apache.REEF.Common.Files
+{
+    public class ResourceHelper
```

Comment:
Please add class-level documentation. Application authors might mistake this with a helper for *their* resources to be copied to the Driver. On a related note: Does this have to be `public`?